app/controllers/auth.rb

```diff
@@ -0,0 +1,19 @@
+post '/authentication/register_user' do
+  if params[:password] == params[:password_confirm]
+    user = User.create(username: params[:username], password: params[:password])
```

Comment:
Probably want to check here if User.create succeeded, otherwise you'll store a nil user id in your session.